labelbox/utils.py

```diff
@@ -80,4 +86,14 @@ def format_iso_datetime(dt: datetime.datetime) -> str:
     Formats a datetime object into the format: 2011-11-04T00:05:23Z
     Note that datetime.isoformat() outputs 2011-11-04T00:05:23+00:00
     """
-    return dt.strftime(ISO_DATETIME_FORMAT)
\ No newline at end of file
+    return dt.strftime(ISO_DATETIME_FORMAT)
```

Comment:
Does this work if we're not backporting anymore? I see `format_iso_datetime` is still being used

Reply:
So one thing I will do is run a script for python versions 3.7-9 locally and I will use logic -pre-backport, backport and post-backport to compare results. This will be based on the original PR that introduced backport: https://github.com/Labelbox/labelbox-python/pull/1045/files. My work here was done in the context of that PR

Reply:
I did not change the function name, just provided a different implementation based on the dateutils package

Reply:
A very good point and I am glad you have asked
This function uses `strftime` and I have tested it and found working the same with and without backport, so I have decided to keep it in light of the immediate goal of this story

HOWEVER I have raised it in this PR this function does not convert timezones into UTC correctly... I was hesitant to address it as a part of this story since no one has ever flagged this as an issue before

LMK, happy to address it here or as a separate ticket